process/multiuser_posix.go

```diff
@@ -27,21 +29,36 @@ func NewPlatformData(currentUser bool) (pd *PlatformData, err error) {
 func TaskUserPlatformData() (pd *PlatformData, err error) {
 	user, err := runtime.InteractiveUsername()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Could not determine interactive username: %v", err)
+	}
+
+	id := func(idName string, idOpts string) (uint32, error) {
```

Comment:
The way this is called is confusing to me because it looks like `"UID"` or `"GID"` are being passed as commands to the `id` command itself. I'd almost rather have separate functions split out, as clunky as that sounds. The `host` package could be a good place for `UserID()` and `UserGID()` or similar.

Reply:
I see what you mean. I guess I could pass the entire command through, rather than just the opts. It felt clunky to me too when I wrote the method, but I also disliked copy/pasting the code when it was essentially the same. 

Maybe if I pass the entire command through it will look cleaner...